code-challenges.js

```diff
@@ -9,40 +9,149 @@
 
+var myForNumbers1 = [3, 7, 0, 6, -9]
+var myForNumbers2 = [2, 5, -8, 20, 32]
 
+// 
+// //the loop that is created should take each number individually and multiply by five
 
+//create a function that is called timesFive
+//takes in an array 
+const timesFive = (array) => {
+    //create an empty array for the values that will be multiplied by five
+    let timedFive = []
+    //iterates through each index 
+    for(let i=0; i<array.length; i++){
+        //multiplies each value by five and adds it to the empty timedFive array
+        timedFive.push(array[i] * 5)
+    }
+    //returns the new array with all the values timed by five
+    return timedFive
+}
 
+console.log(timesFive(myForNumbers1))
+console.log(timesFive(myForNumbers2))
 
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
-// Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
-var myForNumbers1 = [3, 7, 0, 6, -9]
-var myForNumbers2 = [2, 5, -8, 20, 32]
 
 
 
+// // --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
 
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
-
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+//create a function named multiFiveMap
+//takes in an array 
+const multiFiveMap = (array)=>{
+    // iterates through each index and creates a new array with the multiplied values
+    return array.map(value => {
+        //multiplies each value by five
+        return value * 5
+    })
+}
 
 
+console.log(multiFiveMap(myMapNumbers1))
+console.log(multiFiveMap(myMapNumbers2))
 
 
-// --------------------4) Create a function that removes all the vowels from a string.
-// Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
+// // --------------------4) Create a function that removes all the vowels from a string.
+// // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+//-----------------------------------------------------------------------------------------------------------
+//This was my original answer (#4 attempt one) This code worked, but if felt like a work around of what we had been doing in class.
+// I did a search with the google machine and found the ".replace(/[aeiou]/ig,'')" function. My understanding of this function is that it appears to identify the letters is the brackets and then replaces them with nothing which is why the '' is also in the parenthesis. Tested by putting 'R' into the space and a bunch of 'R's showed up in in the string
```

Comment:
nice research. Use of regex is good but it also is 1) difficult for developers to understand because they have to know what the regex is doing 2) for simple problems it's almost like using a tractor to plant a seed 

Some things to note about regex: 
`[aeiou]` indicates that it should match each literal character provided within in `[]`
`ig` are flags (`i` stands for case insensitive which is why it matches for capitalized letters and `g` stands for global matching: look for all instances of those characters in a string. without the `g` flag it will only match on the first vowel in the string and stop trying to match).

Correct, the second argument in .replace() is whatever you want the character to be replaced with